configure.ac

```diff
@@ -12,7 +12,7 @@ dnl Set to c:r+1:a if the interface is unchanged.
 AC_SUBST([libcdd_version_info], [0:0:0])
 
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE([1.14.1])
+AM_INIT_AUTOMAKE([1.14.1 foreign tar-ustar])
```

Comment:
Why do we want foreign and why do we want tar-ustar?

At least this appears to be unrelated to the issue that this PR is about?

Reply:
Straight from the manual: “The ‘foreign’ option tells Automake that this package will not
follow the GNU Standards.  GNU packages should always distribute [and automake would enforce that] additional files such as ‘ChangeLog’, ‘AUTHORS’". As ChangeLog NEWS etc. is effectively empty (the content really is in other files), it's only consequent to delete the extraneous files.

Reply:
Also mentioned in the commit message =)

The "ustar" format is the most portable. The automake documentation explains that, and one of our developers did an in-depth study to confirm it: https://dev.gentoo.org/~mgorny/articles/portability-of-tar-features.html

Reply:
Oh, and yeah, it's unrelated to the title of the PR. But I was mucking around with autotools already and saw the opportunity for a small improvement.